emulators/xhyve/files/cast-int.patch

```diff
@@ -0,0 +1,44 @@
+commit 55e2b427882624157cbce08a211c5af83043bb2f
```

Comment:
Could you upstream this as well?  I didn't see it when quickly looking through the pending pull requests upstream last night (I could have missed it).

Reply:
@jeremyhu I'm usually send to upstream all patches that I'm making. For example this one is here: https://github.com/machyve/xhyve/pull/219

Reply:
BTW, it is quite new. Because without this patch I can compile it on Big Sur until 11.6.